book/README.md

```diff
@@ -1,38 +1,18 @@
 ## The Turing Way Book
 
-This is the README file for _The Turing Way_ book hosted online at https://book.the-turing-way.org/.
-For the README file of the main repository please [follow this link](https://github.com/the-turing-way/the-turing-way/blob/main/README.md).
+This is the README file for _The Turing Way_ book hosted online at [book.the-turing-way.org](https://book.the-turing-way.org).
+For the README file of the main repository please [follow this link](../README.md).
 
-All the text for each chapter of the `book` lives inside the folder `./website` directory.
-All figures associated to the chapters are stored in and linked from the `./website/figures` directory.
-Everything else is in the `website/` directory.
+All the text for each chapter of the `book` lives inside the [`website`](./website/) directory.
+All figures associated to the chapters are stored in, and linked from, the [`figures`](./figures/) directory.
 
 ### Configuration
 
-- The table of contents (TOC) defines the order of chapters as they appear in the book.
-To change the TOC, please edit the `website/myst.yml` file with correct information on filenames and their relative locations in this repository.
-Documentation on controlling the TOC structure can be found on the [Jupyter Book website](https://jupyterbook.org/customize/toc.html).
-- Same applies for more general configuration using `website/myst.yml`.
-Documentation on configuring book settings can be found on the [Jupyter Book website](https://jupyterbook.org/customize/config.html).
+The book's configuration is defined in [`website/myst.yml`](./website/myst.yml). The possible fields for this file are explained [in the MystMD documentation](https://mystmd.org/guide/frontmatter#in-a-myst-yml-file).
 
-### Deploying
+### Building
 
-The site is built automatically using the two directories `book` and `book/website`.
-All of the dependencies are specified in `dependencies.in`, which is transformed into the lock file `dependencies.lock` using `pip-compile`.
-
-Instructions for how to build a Turing Way book locally can be found in [_The Turing Way_'s Community Handbook](https://book.the-turing-way.org/community-handbook/local-build).
-
-#### On Netlify
```

Comment:
I think this might be the only place we document our use of Netlify besides redirects page, and the toml file itself. 
This probably isn't the best place for it, but we might want to keep it in until we're replacing it with something else - what do you think @JimMadge?